demo/theme-vaadin-input-md.html

```diff
@@ -0,0 +1,39 @@
+<dom-module id="theme-vaadin-input-md">
+  <template>
+    <style>
+      html {
+        --vaadin-input-input: {
+          outline: none;
+          border: none;
+          background: transparent no-repeat;
+          background-image:
+           linear-gradient(to bottom, var(--primary-color, deepskyblue), var(--primary-color, deepskyblue)),
+           linear-gradient(to bottom, var(--secondary-text-color, silver), var(--secondary-text-color, silver));
+          background-size: 0 2px, 100% 1px;
+          background-position: 50% 100%, 50% 100%;
+
+          font-family: sans-serif;
+          font-size: 16px;
+          @apply --paper-font-subhead;
```

Comment:
I’m not sure we want to depend on paper-styles in our themes.At least not yet, since there’s so little that we need from there for this input.

Also, this dependency is not declared or imported anywhere, it just assumes paper-styles/typography is present on the page.